docs/readthedocs/source/doc/LLM/Overview/install_gpu.md

```diff
@@ -127,9 +127,49 @@ BigDL-LLM for GPU supports on Linux has been verified on:
 
       * Step 2: Download and install `Intel® oneAPI Base Toolkit <https://www.intel.com/content/www/us/en/developer/tools/oneapi/base-toolkit-download.html>`_ with version 2024.0. OneDNN, OneMKL and DPC++ compiler are needed, others are optional.
 
-        .. seealso::
+      Intel® oneAPI Base Toolkit 2024.0 install methods:
+
+      .. tabs::
+         .. tab:: Offline installer
+         
+            We recommend you to use the offline package installer. You can specify the install path during the install process.
+
+            .. code-block:: bash
+            
+               wget https://registrationcenter-download.intel.com/akdlm/IRC_NAS/20f4e6a1-6b0b-4752-b8c1-e5eacba10e01/l_BaseKit_p_2024.0.0.49564_offline.sh
+               sudo sh ./l_BaseKit_p_2024.0.0.49564_offline.sh
+
+            .. note::
+
+                  You can also modify the installation or uninstall the package by re-running the installer
+
+                  .. code-block:: bash
+                  
+                     sudo sh ./l_BaseKit_p_2024.0.0.49564_offline.sh
```

Comment:
can we use this command to uninstall?

Reply:
This command can be used. It launches an interactive UI for install/modify/uninstall.
However, I will modify the uninstall instruction to run the copy of the installer created in the oneapi folder instead. This is probably more convenient.